### DISCUSSION TITLE
How to run chromedriver ENTIRELY with SSL verification disabled?

### DISCUSSION THREAD

**Discussion by andburch** [ultrafunkamsterdam/undetected-chromedriver#1168] [Q&A]
My job is behind a crazy firewall, and the only way to get any applications to run is by entirely turning off SSL verification. (The firewall system handles all this for us, I guess). 

HOWEVER, before I can even l visit a website, the undetected-chromedriver makes HTTP requests during initialization and I can't change their settings, as seen in [`fetch_release_number()`](https://github.com/ultrafunkamsterdam/undetected-chromedriver/blob/bf7dcf8b5713020de7454844fb80036b8c456503/undetected_chromedriver/patcher.py#L134) via the `urlopen()` call.  **Is there any way to force those `urlopen()` calls to ALSO ignore SSL verification?** Like, is there a way to set a global "don't SSL verify" option?

**Comment by tromboris** [comment]
The info I found for this particular scenario is to import SSL, and use the following line of code before instantiating your driver:
ssl._create_default_https_context = ssl._create_unverified_context

I have used it successfully in projects.
From what I read about it afterward, it is only to be used with great caution because it really will get rid of the need for any SSL verification whatsoever.
